src/components/ProgramCertificateModal/messages.js

```diff
@@ -0,0 +1,81 @@
+import { defineMessages } from '@edx/frontend-platform/i18n';
+
+const messages = defineMessages({
+  certificateModalTitle: {
+    id: 'credentials.modal.title',
+    defaultMessage: 'Verifiable credential',
+    description: 'Title of a dialog.',
+  },
+  certificateModalCloseBtn: {
+    id: 'credentials.modal.close.button',
+    defaultMessage: 'Close modal window',
+    description: 'Label on button to close a dialog.',
+  },
+  certificateModalCloseMobileBtn: {
+    id: 'credentials.modal.close.mobile.button',
+    defaultMessage: 'Cancel',
+    description: 'Label on button to close a dialog.',
+  },
+  certificateModalMobileTitle: {
+    id: 'credentials.modal.mobile.title',
+    defaultMessage: 'To download a verifiable credential to your mobile wallet application, please follow the instructions below.',
+    description: 'Text for a mobile dialog of the program certificate.',
+  },
+  certificateModalAppStoreBtn: {
+    id: 'credentials.modal.instruction.appStore.button',
+    defaultMessage: 'Download the mobile app from the Apple App Store',
+    description: 'The label for the link to download the apple version of the app.',
+  },
+  certificateModalGooglePlayBtn: {
+    id: 'credentials.modal.instruction.googlePlay.button',
+    defaultMessage: 'Download the  mobile app from the Google Play',
+    description: 'The label for the link to download the google version of the app.',
+  },
+  certificateModalInstructionTitle: {
+    id: 'credentials.modal.instruction.title',
+    defaultMessage: 'Download and install the app on your smartphone.',
+    description: 'Title text of the instructions.',
+  },
+  certificateModalInstructionStep1: {
+    id: 'credentials.modal.instruction.step1',
+    defaultMessage: 'Sign up for the app to identify yourself.',
+    description: 'Text of step of the instructions.',
+  },
+  certificateModalInstructionStep2: {
+    id: 'credentials.modal.instruction.step2',
+    defaultMessage: 'Open the application and select the option scan the QR code. Scan the provided code.',
+    description: 'Text of step of the instructions.',
+  },
+  certificateModalInstructionStep3: {
+    id: 'credentials.modal.instruction.step3',
+    defaultMessage: 'Follow this instructions below to get verifiable credential:',
```

Comment:
Not sure what the design suggests, but I think this should say "Follow the instructions below to get the verifiable credential"